### PR TITLE
rewrite app-features and app-works with sass and bem

### PR DESCRIPTION
**Pull request by vladhybalo**

### CODE REVIEW

File: sass/appFeatures.sass [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+.app-feature
+  display: flex
+  flex-direction: column
+  align-items: center
+
+  &_center
+    display: flex
+    flex-direction: column
+    align-items: center
+    margin-top: 200px
```
Comment (LenaAviles) [inline sass/appFeatures.sass]: avoid using margin-top

File: sass/appFeatures.sass [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    width: 100%
+    display: flex
+
+
+.app-feature
+  display: flex
+  flex-direction: column
+  align-items: center
+
+  &_center
```
Comment (LenaAviles) [inline sass/appFeatures.sass]: please use international notation for modifiers `--`

File: sass/mixins.sass [hunk 3]
```diff
@@ -0,0 +1,11 @@
+%align_cross_axis
+  display: flex
+  align-items: center
+
+%align_main_axis
+  display: flex
+  justify-content: center
+
+%open-sans
```
Comment (LenaAviles) [inline sass/mixins.sass]: it's meaningless mixin

File: sass/appFeatures.sass [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  &__image
+    width: 100%
+    max-width: 61px
+    height: auto
+    object-fit: contain
+    max-height: 60px
+
+  &__header
+    @extend %open-sans
+    color: #606060
```
Comment (LenaAviles) [inline sass/appFeatures.sass]: it's better to keep color in vars

File: sass/appWorks.sass [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    display: flex
+    width: 100%
+
+    &-image-wrap
+      width: 100%
+      max-width: 20%
+      @extend %align_cross_axis
+      margin-right: 1rem
+
+    &-image
```
Comment (LenaAviles) [inline sass/appWorks.sass]: please, don't use & this way,  I want to see whole element's name

File: index.html [hunk 6]
```diff
@@ -14,232 +14,232 @@
...
-                </div>
+<section class="first-view">
+    <nav class="navbar">
+        <div class="container-fluid">
+            <a class="navbar-brand" href="#">Prototype</a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive"
+                    aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon "></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarResponsive">
```
Comment (LenaAviles) [inline index.html]: don't use camelCase for id and files names, use kebab-case instead

File: index.html [hunk 7]
```diff
@@ -14,232 +14,232 @@
...
+</section>
+<main class="container-wrap">
     <div class="container">
         <section class="app-features">
-            <div class="row">
-                <div class="col-sm d-flex flex-column align-items-center">
-                    <img src="./img/cloud.png" class="app-feature__item__icon" alt="cloud icon">
-                    <h3 class="app-feature__item__header open-sans">
+            <div class="app-features__inner">
+                <div class="app-feature">
```
Comment (LenaAviles) [inline index.html:78]: it's would be better `app-features__item`

File: sass/appFeatures.sass [hunk 8]
```diff
@@ -0,0 +1,43 @@
+@import "variables"
+@import "mixins"
+
+.app-features
+  padding-top: 5rem
```
Comment (ArtMrsh) [inline sass/appFeatures.sass]: --

File: sass/appWorks.sass [hunk 9]
```diff
@@ -0,0 +1,41 @@
+@import "./mixins"
+@import "./variables"
+
+.app-works
+  width: 100%
+  @extend %align_cross_axis
+  padding: 40px 20px
```
Comment (ArtMrsh) [inline sass/appWorks.sass]: ---

**Pull request merged**